### ISSUE TITLE
Post most recent event(s) on homepage

### ISSUE THREAD

**Issue by nonlinear** [GothamSass/chrysler#28]
Homepage should list upcoming event(s). We will likely just have one event, but the design should account for multiple events. 
- [ ] @nonlinear: Sketch/design for the homepage event component
- [ ] @rmjames: Write the JS to only pull the events for current month

**Comment by misscs** [comment]
Notes: I've updated this task to be more specific and added two todos.

@nonlinear I think it would be best if you could start by doing some sketches for the homepage event component. As we discussed we can assume that we'll usually have one event a month, but sometimes we may have more than one. 

Secondly, we need to decide on the logic for displaying these events. Is it current month or one month out from today's date? And if there are multiple events for a month, what's the criteria for deciding which appears first. 

In short, let's do a bit of planning on this one before diving into code and ensure #46 is resolved first
